### DISCUSSION TITLE
Having issues with Z2M Groups

### DISCUSSION THREAD

**Discussion by drc13a** [Koenkk/zigbee2mqtt#22942] [Devices]
I'm having trouble getting my device groups functioning and not sure where to start on troubleshooting it.

2 groups I have created with 2 zigbee lamp bulbs (TuYa TS0502B) work perfectly

Any other groups I create with my light switches (combination of Moes ZS-EUB_2gang and Moes ZS-EUB_1gang) I can't get working.

The group is created without issues, I'm able to toggle the group switch and the appropriate switches correctly toggle in the dashboard but the actual lights don't activate. These same devices work perfectly if I toggle them individually on the dashboard or when grouped using standard HA groups just NOT when I use the z2m group toggle for them.

Happy to troubleshoot further if I can be pointed in the right direction.

Thanks in advance

**Comment by kth1** [comment]
I'm having the same trouble with device groups ... restarting Z2M and unplugging my SONOFF stick are without effect.
After deleting the group and setting up a new one it works fine for a few days, but now the same effect shows again ... slides change in the dashboard, but without effect on the room lights.
Is there perhaps a solution that I have overlooked?

**Comment by Action2103** [comment]
Totally same issue I have. Two Aqara Zigbee switches grouped in Z2MQTT. Once I toggle the newly created group switch the devices's states are changing but physically the switches are not triggered. The same if I create a scene inside the MQTT group, one scene is ON, second scene is OFF, the same thing is happening. Sates are following the command, but switches in reality not. If I group them in HA helper they are working without any issues.

**Comment by MeDM91** [comment]
Same issue here. I have 3 perfectly fine working groups in z2m. Yesterday I created a 4th one and it just works 10% of the time. My motion sensor turns on the group in HA, but the lights stay off most of the time.

**Comment by vellad1** [comment]
Same issue here.  Using Z2M 2.6.3.  A group to 2 light switches (both MOES Single Gang.   I switch them on from HA,  the state is reflected in Z2M but the actual lights don't come on.

**Comment by uzairfarooq** [comment]
Same issue

**Comment by andrei-lazarov** [comment]
Sleepy **end devices** don't support groups (Zigbee limitation). Only **routers** do as far as I know
